### PR TITLE
Add Oauth

### PR DESCRIPTION
**Pull request by thathert**
Gets the authorize button working on the swagger page.

### CODE REVIEW

File: src/main/java/bio/terra/SwaggerDocumentationConfig.java [hunk 1]
```diff
@@ -1,21 +1,84 @@
...
+
+    public String[] getScopes() {
+        return scopes.clone();
+    }
+
+    public void setScopes(String[] scopes) {
+        this.scopes = scopes.clone();
+    }
+
     ApiInfo apiInfo() {
```
Comment (ddietterich) [inline src/main/java/bio/terra/SwaggerDocumentationConfig.java:82]: Just to clarify, adding the `@ConfigurationProperties("oauth")` provides the prefixes coming _from_ the config file, but does not affect how these beans are references from inside the code. Right?

**Review by ddietterich — APPROVED** [review]
LGTM - very tidy :)

File: src/main/java/bio/terra/SwaggerDocumentationConfig.java (hunk 1 above)
Comment (thathert) [reply src/main/java/bio/terra/SwaggerDocumentationConfig.java:82]: Right, this is the way to tell Spring to look into application.properties and match anything with the prefix `oauth.` or to match any environment variable that start with `OAUTH_`.

**Review by ahaessly — APPROVED** [review]
👍

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**